### PR TITLE
feat: add navigation

### PR DESCRIPTION
**Pull request by Floran-mtte**

### CODE REVIEW

**Review by codebynao — APPROVED** [review]

File: src/layouts/Footer.jsx [hunk 1]
```diff
@@ -1,12 +1,141 @@
...
 
 const Footer = () => {
+  const [collapse, setCollapse] = useState(false);
+  const [collapseIcon, setCollapseIcon] = useState("chevron-left");
+
+  const collapseSideBar = () => {
+    let sideBar = document.getElementsByClassName("nav");
+    let sideBarText = document.getElementsByClassName("tab__text");
+    let logo = document.getElementsByClassName("logo__app");
+    let collapseText = document.getElementsByClassName("collapse__text");
```
Comment (itsabdessalam) [inline src/layouts/Footer.jsx]: Here, you browse the DOM several times, you can either store `document` in a variable and reuse it, or use `useRef` hook which gives you direct access to your current parent element and its children

File: src/layouts/Footer.jsx [hunk 2]
```diff
@@ -1,12 +1,141 @@
...
+
+    if (!collapse) {
+      collapseText[0].style.display = "none";
+      setCollapseIcon("chevron-right");
+      for (let i = 0; i < sideBarText.length; i++) {
```
Comment (itsabdessalam) [inline src/layouts/Footer.jsx]: You add complexity by manipulating the styles inside a loop, why not just add a parent css class and hide and display elements through this class?

File: src/layouts/Footer.jsx [hunk 3]
```diff
@@ -1,12 +1,141 @@
...
+
+      sideBar[0].style.width = "50px";
+      setCollapse(true);
+      return;
+    }
+
+    sideBar[0].style.width = "20%";
+    collapseText[0].style.display = "flex";
+    setCollapseIcon("chevron-left");
+    for (let i = 0; i < sideBarText.length; i++) {
```
Comment (itsabdessalam) [inline src/layouts/Footer.jsx]: Same as https://github.com/itsabdessalam/taskee/pull/3/files#r528603644

File: src/components/TabItem.jsx [hunk 4]
```diff
@@ -31,14 +30,30 @@ const StyledTabItem = styled.div`
     font-size: 10px;
     margin-top: 2px;
   }
+
+  @media (min-width: ${({ theme }) => theme.screens.md}) {
+    height: 5%;
```
Comment (itsabdessalam) [inline src/components/TabItem.jsx]: I'm not a big fan of using percentages like this, especially in our case

**Review by itsabdessalam — CHANGES_REQUESTED** [review]
I'v added some comments

Also, for your information 

image

image

When we switch to a mobile device after collapsing the sidebar, tab bar doesn't look good

image

**Review by itsabdessalam — APPROVED** [review]
LGTM!

**Pull request merged**